packages/vuetify/src/util/helpers.ts

```diff
@@ -322,6 +322,28 @@ type Writable<T> = {
   -readonly [P in keyof T]: T[P]
 }
 
+/*
+ Collects the VNodes insdie the provided slots that holds the name specified
+Note: for some reason, recursively returning array and adding the content via concat did not work, therefore passing the array as parameter!
+ */
+export function addNodesInSlots (name: string, slots: Slots, nodes?: VNode[]) {
```

Comment:
I'd like @KaelWD's opinion on this. Early on we decided against VNode parsing in v3 which is why `v-tabs` is the way it is now.